osf/migrations/__init__.py

```diff
@@ -126,6 +129,25 @@ def update_permission_groups(sender, verbosity=0, **kwargs):
         update_provider_auth_groups(verbosity)
 
 
+def update_storage_regions(sender, verbosity=0, **kwargs):
+    if getattr(sender, 'label', None) == 'osf':
+        ensure_default_storage_region()
+
+
+def ensure_default_storage_region():
+    osfstorage_config = apps.get_app_config('addons_osfstorage')
+    Region = apps.get_model('addons_osfstorage', 'Region')
+    Region.objects.get_or_create(
+        _id=DEFAULT_REGION_ID,
+        defaults={
+            'name': DEFAULT_REGION_NAME,
+            'waterbutler_credentials': osfstorage_config.WATERBUTLER_CREDENTIALS,
+            'waterbutler_settings': osfstorage_config.WATERBUTLER_SETTINGS,
+            'waterbutler_url': settings.WATERBUTLER_URL
+        }
+    )
```

Comment:
Nit-pick: I understand that this is the same as the old code in the original migration, but I think pulling `name` out of the `defaults` param is more consistent with the [Doc 1.11](https://docs.djangoproject.com/en/1.11/ref/applications/#django.apps.AppConfig) since the model is [unique](https://github.com/CenterForOpenScience/osf.io/blob/develop/addons/osfstorage/models.py#L488) on (`_id` and `name`).

> Any keyword arguments passed to get_or_create() — except an optional one called defaults — will be used in a [get()](https://docs.djangoproject.com/en/1.11/ref/models/querysets/#django.db.models.query.QuerySet.get) call. If an object is found, get_or_create() returns a tuple of that object and False.

```python
Region.objects.get_or_create(
        _id=DEFAULT_REGION_ID,
        name=DEFAULT_REGION_NAME,
        defaults={
            'waterbutler_credentials': osfstorage_config.WATERBUTLER_CREDENTIALS,
            'waterbutler_settings': osfstorage_config.WATERBUTLER_SETTINGS,
            'waterbutler_url': settings.WATERBUTLER_URL
        }
    )
```

Reply:
Fixed in https://github.com/CenterForOpenScience/osf.io/pull/9965/commits/4cbcf633cb55cdca52bb2c2cd570e9f4add97f14